pallets/clients-info/Cargo.toml

```diff
@@ -7,38 +7,31 @@ edition = "2021"
 [dependencies]
 codec = {package = "parity-scale-codec", version = "3.1.5", default-features = false, features = ["derive", "max-encoded-len"]}
 scale-info = {version = "2.2.0", default-features = false, features = ["derive"]}
-serde = {version = "1.0.130", default-features = false, features = ["derive"], optional = true}
 
 # Substrate dependencies
-sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
```

Comment:
Note: `sp-runtime` is a dependency found in **`frame-support`**. We don't need to add `sp-runtime` as a dependency.